src/UhdmAst.cpp

```diff
@@ -2149,18 +2148,9 @@ AstNode* visit_object(vpiHandle obj_h, UhdmShared& shared) {
         }
         }
 
-        if (alwaysType != VAlwaysKwd::ALWAYS_COMB && alwaysType != VAlwaysKwd::ALWAYS_LATCH) {
-            // Handled in vpiEventControl
-            AstNode* always;
-
-            visit_one_to_one({vpiStmt}, obj_h, shared, [&](AstNode* node) { always = node; });
-            return always;
-        } else {
-            // Body of statements
-            visit_one_to_one({vpiStmt}, obj_h, shared, [&](AstNode* node) { body = node; });
-        }
-
-        return new AstAlways(make_fileline(obj_h), alwaysType, senTree, body);
+        // Body of always statement
+        visit_one_to_one({vpiStmt}, obj_h, shared, [&](AstNode* nodep) { bodyp = nodep; });
+        return new AstAlways(make_fileline(obj_h), alwaysType, nullptr, bodyp);
     }
     case vpiEventControl: {
         AstSenItem* senItemRoot = nullptr;
```

Comment:
Seems like a good opportunity to:
1. Fix the naming convention.
2. Refactor this to a `process_*` function.

Reply:
Done